.github/workflows/conformance.yml

```diff
@@ -299,6 +299,11 @@ jobs:
         docker-compose -f ./.github/infrastructure/docker-compose-postgresql.yml -p postgresql up -d
       if: contains(matrix.component, 'postgresql')
 
+    - name: Start postgres
```

Comment:
No, please remove. You can just update the name above and make it `postgres` which will match `postgresql` also :)

Reply:
Yes. I will remove it. I am just testing a few things.